src/docs/component/ProgressTracker.vue

```diff
@@ -0,0 +1,63 @@
+F<template>
+  <div class="aui-group">
+    <div class="aui-item">
+      <aui-tabs>
+        <aui-tab name="Example">
+          <aui-progress-tracker>
+            <aui-progress-tracker-step>First Step</aui-progress-tracker-step>
+            <aui-progress-tracker-step>Second Step</aui-progress-tracker-step>
+            <aui-progress-tracker-step active>Third Step</aui-progress-tracker-step>
+            <aui-progress-tracker-step>Fourth Step</aui-progress-tracker-step>
+          </aui-progress-tracker>
+
+          <aui-progress-tracker>
+            <aui-progress-tracker-step active><a href="#">First Link</a></aui-progress-tracker-step>
+            <aui-progress-tracker-step><a href="#">Second Link</a></aui-progress-tracker-step>
+            <aui-progress-tracker-step><a href="#">Third Link</a></aui-progress-tracker-step>
+            <aui-progress-tracker-step><a href="#">Fourth Link</a></aui-progress-tracker-step>
+          </aui-progress-tracker>
```

Comment:
Please add examples for more and less than 4 steps. (you can change one of the existing ones too)